caching-system/memcached/memcached.def

```diff
@@ -0,0 +1,18 @@
+Bootstrap:library
+From: ubuntu:18.04
```

Comment:
Using Ubuntu 18.x this will not run on RHEL 6. Do we want to put in a comment about that restriction?